cppapi/server/jpeg_mmx/CMakeLists.txt

```diff
@@ -20,21 +20,29 @@ if(WIN32)
         target_compile_definitions(jpeg_mmx_objects_sta PRIVATE _64BITS)
     else()
         target_compile_options(jpeg_mmx_objects_dyn PRIVATE -O0)
-        target_compile_definitions(jpeg_mmx_objects_dyn PRIVATE JPG_USE_ASM)
-        target_compile_options(jpeg_mmx_objects_sta PRIVATE -O0)
-        target_compile_definitions(jpeg_mmx_objects_sta PRIVATE JPG_USE_ASM)
+	target_compile_options(jpeg_mmx_objects_sta PRIVATE -O0)
+	if(TANGO_JPEG_MMX)
+            target_compile_definitions(jpeg_mmx_objects_dyn PRIVATE JPG_USE_ASM)
+            target_compile_definitions(jpeg_mmx_objects_sta PRIVATE JPG_USE_ASM)
+	endif()
     endif()
```

Comment:
Indentation is very inconsistent here. In just 10 lines of code we indent blocks by two, four and six spaces.
This is not a major issue, but please consider enabling auto-indentation in your editor for future PRs.

Reply:
@mliszcz You are right. I think we should also have some rough coding guidelines for CMAKE. There is even https://github.com/cheshirekow/cmake_format although.

Reply:
Agree, 6 spaces is clearly a misprint.  How many do we want? 2? 4? 8?